src/nvhttp.cpp

```diff
@@ -217,48 +222,72 @@ namespace nvhttp {
       return;
     }
 
-    pt::ptree root;
+    pt::ptree tree;
     try {
-      pt::read_json(config::nvhttp.file_state, root);
+      pt::read_json(config::nvhttp.file_state, tree);
     }
     catch (std::exception &e) {
       BOOST_LOG(error) << "Couldn't read "sv << config::nvhttp.file_state << ": "sv << e.what();
 
       return;
     }
 
-    auto unique_id_p = root.get_optional<std::string>("root.uniqueid");
+    auto unique_id_p = tree.get_optional<std::string>("root.uniqueid");
     if (!unique_id_p) {
       // This file doesn't contain moonlight credentials
       http::unique_id = uuid_util::uuid_t::generate().string();
       return;
     }
     http::unique_id = std::move(*unique_id_p);
 
-    auto device_nodes = root.get_child("root.devices");
-
-    for (auto &[_, device_node] : device_nodes) {
-      auto uniqID = device_node.get<std::string>("uniqueid");
-      auto &client = map_id_client.emplace(uniqID, client_t {}).first->second;
-
-      client.uniqueID = uniqID;
+    auto root = tree.get_child("root");
+    client_t client;
+    client.uniqueID = "0123456789ABCDEF"s;
```

Comment:
Why is this hard coded?

Reply:
I wasn't sure if the `uniqueID` is being used elsewhere in Sunshine. I had refactored out most of the code related to mapping certificates to `uniqueID`s, but left this (hard-coded to Moonlight's) just in case. If it's not needed, then I believe `uniqueID` can be removed from `client_t` entirely.

Reply:
@cgutman pinging you since you're the most familiar with this.

Reply:
You can remove it. Nvidia used it to identify separate clients, but it caused more issues than it solved (preventing a game started on one device from being resumed on another). That's why Moonlight just hardcodes a single value everywhere.

Reply:
Removed 👍 (https://github.com/LizardByte/Sunshine/pull/2042/commits/5ff80ada0b93dd8527c0c7e20e42e9f75065b0f3)